src/components/ChartCard/Chart.tsx

```diff
@@ -27,6 +27,7 @@ type IChartProps = { setId: (Dispatch<SetStateAction<number>>), total: number; }
 )
 
 const UserExpenseChart: FC<IChartProps> = ({ categories, members, total, setId }) => {
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency);